From 2865ecfe5e02f8895c555396993b49b3b08a3550 Mon Sep 17 00:00:00 2001
From: Chet Murthy <chetsky@gmail.com>
Date: Fri, 15 Jan 2021 12:35:50 -0800
Subject: [PATCH] fix META file generation (DESTDIR) issue; Fix #75

---
 Makefile     | 2 +-
 etc/Makefile | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index c9adcc46..7748ddd6 100644
--- a/Makefile
+++ b/Makefile
@@ -51,7 +51,7 @@ depend:
 
 local-install::
 	$(RM) -rf local-install && mkdir -p local-install/lib/ocaml
-	$(MAKE) DESTDIR=`pwd`/local-install/ LIBDIR=lib BINDIR=bin MANDIR=man install
+	$(MAKE) DESTDIR= LIBDIR=`pwd`/local-install/lib BINDIR=`pwd`/local-install/bin MANDIR=man install
 
 install:
 	$(NOVERBOSE) if test -z "$(LIBDIR)"; then \
diff --git a/etc/Makefile b/etc/Makefile
index ef4593cc..f8913866 100644
--- a/etc/Makefile
+++ b/etc/Makefile
@@ -106,7 +106,7 @@ install:
 	-$(MKDIR) "$(DESTDIR)$(LIBDIR)/$(CAMLP5N)" "$(DESTDIR)$(BINDIR)"
 	$(RM) -f "$(DESTDIR)$(LIBDIR)/$(CAMLP5N)/pa_lex.cmo"
 	cp $(OBJS) $(INSTALL_CMI) "$(DESTDIR)$(LIBDIR)/$(CAMLP5N)/."
-	sed -e "s|@VERSION@|$(VERSION)|" -e "s|@CAMLP5DIR@|$(DESTDIR)$(LIBDIR)/$(CAMLP5N)|" META.tpl > META
+	sed -e "s|@VERSION@|$(VERSION)|" -e "s|@CAMLP5DIR@|$(LIBDIR)/$(CAMLP5N)|" META.tpl > META
 	cp META "$(DESTDIR)$(LIBDIR)/$(CAMLP5N)/."
 	cp lib.sml "$(DESTDIR)$(LIBDIR)/$(CAMLP5N)/."
 	cp $(CAMLP5N)o$(EXE) $(CAMLP5N)sch$(EXE) "$(DESTDIR)$(BINDIR)/."
